java-checks/src/test/files/checks/NestedSwitchStatementCheck.java

```diff
@@ -1,5 +1,3 @@
-import java.util.List;
```

Comment:
Do not remove that. You still need `java.util.List`, you are declaring it line 56! That's `ArrayList` which was not imported, and also useless.